src/renderer/scss/component/_button.scss

```diff
@@ -1,232 +1,160 @@
 .btn {
-  min-width: var(--btn-height);
-  height: var(--btn-height);
-
-  align-items: center;
-  background-color: $lbry-teal-5;
-  border: none;
-  border-radius: var(--btn-radius);
-  color: $lbry-white;
-  cursor: pointer;
-  display: flex;
-  fill: currentColor; // for proper icon color
-  font-size: 12px;
-  justify-content: center;
-  padding: 10px;
+  fill: currentColor;
   position: relative;
-  text-decoration: none;
-  transition: all var(--animation-duration) var(--animation-style);
-
-  &:not(:disabled) {
-  }
+}
 
+.btn--alt {
   &:hover {
-    background-color: $lbry-teal-4;
+    text-decoration: underline;
   }
+}
 
-  &:not(.btn--no-padding):not(.btn--link):not(.btn--no-style) {
-    .btn__content {
-      align-items: center;
-      display: flex;
-      padding: 0 8px;
-    }
-  }
+.btn--constrict {
+  @include constrict;
+}
 
-  &.btn--no-padding {
-    .btn__content {
-      padding: 0;
-      display: flex;
-      align-items: center;
-    }
-  }
+.btn--disabled {
+  opacity: 0.3;
+}
 
-  &.btn--alt {
-    &:not(:disabled) {
-      background-color: $lbry-white;
-      color: $lbry-black;
-    }
+.btn--external-link,
+.btn--link {
+  transition: color 0.2s;
 
-    &:disabled {
-      background-color: transparent;
-      color: $lbry-gray-5;
+  &:not(.btn--disabled):not(:hover) {
+    color: $lbry-teal-5;
+
+    html[data-theme='dark'] & {
+      color: $lbry-teal-3;
     }
   }
 
-  &.btn--arrow {
-    color: $lbry-black;
-    width: var(--btn-arrow-width);
+  &:not(.btn--disabled):hover {
+    color: $lbry-teal-3;
 
-    &:disabled {
-      opacity: 0.3;
+    html[data-theme='dark'] & {
+      color: $lbry-teal-4;
     }
   }
+}
 
-  &.btn--danger {
-    background-color: $lbry-red-3;
-  }
-
-  &.btn--link {
-    padding: 0;
-    margin: 0;
-    background-color: inherit;
-    color: $lbry-teal-4;
-    font-size: 1em;
-    border-radius: 0;
-    display: inline;
-    min-width: 0;
-    box-shadow: none;
-    text-align: left;
+.btn--external-link {
+  font-weight: 500;
+  text-align: left;
+}
 
-    &:disabled {
-      color: $lbry-gray-5;
-    }
-  }
+.btn--icon {
+  width: 5rem;
+  height: 5rem;
 
-  &:disabled {
-    cursor: default;
+  background-repeat: no-repeat;
+  background-size: 50%;
+  border-radius: 50%;
+  color: $lbry-white;
+  transition: background-color 0.2s;
 
-    &.btn--primary {
-      background-color: rgba($lbry-black, 0.5);
-    }
+  &:not(:hover) {
+    background-color: rgba($lbry-black, 0.7);
+  }
 
-    &:hover {
-      box-shadow: none;
-    }
+  &:hover {
+    background-color: $lbry-green-3;
   }
 
-  &.btn--external-link {
-    color: $lbry-blue-1;
+  &.btn--play {
+    background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'%3E %3Cg stroke='white' stroke-width='2' fill='white' fill-rule='evenodd' stroke-linejoin='round'%3E %3Cpolygon points='5 21 5 3 21 12'/%3E %3C/g%3E %3C/svg%3E");
```

Comment:
These should stil use the `Icon` component

Reply:
I thought this was going to be resolved after this PR was merged?

Reply:
If we are adding new icons I would prefer they are implemented using the `Icon` component, just so it doesn't slip through the cracks.

Reply:
We had a whole conversation about this. It'd require creating a new repo to replicate the react-feathers module. Should this be done now, or later?

Reply:
It wouldn't require a new repo, that might be a better long-term solution. This could be implemented by something like this:

```
// in component/common/icon

const customIcons = {
      icons.PLAY: <svg>...</svg>,
      icons.EYE: <svg>...</svg>,
      ...
    }
const Icon = customIcons[this.props.icon] || FeatherIcons[this.props.icon];
```

Then we have all of these new custom icons in one place.

Reply:
I wish this was brought up when we had that conversation.

Reply:
My fault. Sorry about that, I thought we were on the same page.